Source/WTF/wtf/MathExtras.h

```diff
@@ -465,6 +465,13 @@ constexpr uint32_t roundUpToPowerOfTwo(uint32_t v)
     return v;
 }
 
+// From https://stackoverflow.com/questions/3407012/rounding-up-to-the-nearest-multiple-of-a-number
```

Comment:
We already have this in WTF, also there are licensing issues with using a StackOverflow answer

Reply:
Updated